example_scenes/opengl.py

```diff
@@ -0,0 +1,86 @@
+from manim import *
+from manim.opengl import *
+import os
+from pathlib import Path
+
+
+class SurfaceExample(Scene):
+    def construct(self):
+        # surface_text = Text("For 3d scenes, try using surfaces")
+        # surface_text.fix_in_frame()
+        # surface_text.to_edge(UP)
+        # self.add(surface_text)
+        # self.wait(0.1)
+
```

Comment:
The purpose of this was to show how the features of the community version differ from those of the upstream version since so much of this example is copy/pasted from that one.

I can remove it if you don't like that idea.

Reply:
I see... then I'm indifferent. It's fine to include but perhaps preface it with a comment saying this feature isn't implemented yet since it just looks like dead code you didn't want to include in the example.

Reply:
A added a comment in.